src/rust/cryptography-x509/src/extensions.rs

```diff
@@ -379,4 +380,13 @@ mod tests {
         assert!(ku.encipher_only());
         assert!(ku.decipher_only());
     }
+
+    #[test]
+    fn test_duplicate_extensions_error_debug() {
```

Comment:
I needed to let `DuplicateExtensonsError` implement the `Debug` trait since I'm using `Extensions::from_raw_extensions` in my unit tests. However, once I do that, coverage is missing for the `#[derive(Debug)]`, so I need a unit test to execute that...

Not sure if there's a better way around this as it's a pretty meaningless test.

Reply:
Yeah, this is an annoying (but understandable) thing about how coverage is collected.

I think https://github.com/pyca/cryptography/pull/9781/files#r1374325340 (or wherever the `Debug` use is introduced) should work to avoid this.

Reply:
[cc0c13e](https://github.com/pyca/cryptography/pull/9781/commits/cc0c13e622b2a432c0856c3cf89d3c34ccccdf8b)